platforms/gke/base/use-cases/federated-learning/terraform/_shared_config/uc_federated_learning_example_cross_device_variables.tf

```diff
@@ -0,0 +1,354 @@
+# Copyright 2024 Google LLC
```

Comment:
You should alphabetize locals, variables, and arguments whenever possible in your Terraform files.